docs/infrastructure_and_maintenance/clustering/ibexa_cloud_and_ddev.md

```diff
@@ -0,0 +1,95 @@
+---
+description: How to run locally an Ibexa Cloud project using DDEV.
+---
+
+# Ibexa Cloud and DDEV
+
+Two ways are available to run an Ibexa Cloud project locally using DDEV:
+
+- [Using the `ddev-platformsh` add-on](#simulate-ibexa-cloud-using-the-ddev-platformsh-add-on).
+- [Like other existing project, without this add-on](#simulate-ibexa-cloud-without-the-platformsh-add-on).
+
+## Simulate Ibexa Cloud using the `ddev-platformsh` add-on
+
+To configure the [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh), you'll need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
+
+`COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variables' separators.
+But the variable must exist for Platform.sh hooks' scripts to work. An `auth.json` file can be used, see [Using an auth.json](install_using_ddev.md#using-an-authjson) for more.
```

Comment:
@dabrt  I wrote "hooks'" because `hooks` is the entry key in .platform.app.yaml. Does it work if I highlight as code?:

```suggestion
`COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variable separators.
But the variable must exist for Platform.sh `hooks` scripts to work. To use an `auth.json` file for this purpose, see [Using an auth.json](install_using_ddev.md#using-an-authjson).
```